src/src/Entity/FormEntity.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\FormEntityRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: FormEntityRepository::class)]
+class FormEntity
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\Column(length: 255, nullable: true)]
+    private ?string $Formulary = null;
```

Comment:
Properties should start with a lowercase letter (so `$formulary`). Also, "formulary" is not a great name for the property - it doesn't hold a form, but just a string with some human-readable rating. I'd prefer `value` (if the entity is called `Rating`) or something similar.